### PR TITLE
VC-1429 - Crash: ExceptionsManagerModule.java line 83 

### PR DESCRIPTION
**Pull request by lumberman**
https://jesusfilmmedia.atlassian.net/browse/VC-1429

### CODE REVIEW

**Comment by lumberman** [comment]
There is an Error in crahlytics:
`TypeError: undefined is not an object (evaluating 'o.default.language.toUpperCase')`
I added extra checks for the lang object to cover it and extra reporting for Crashlytics to be able to see when stuff like this is happening.

File: src/domain/Adventures/AdvLanguageSwitch/index.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const AdvLanguageSwitch = (): ReactElement => {
   }, [currentLang]);
 
   const fillLanguages = (): void => {
-    const appLang = i18next.language.toUpperCase();
+    const appLang = i18next?.language ? i18next.language.toUpperCase() : 'EN';
```
Comment (tataihono) [inline src/domain/Adventures/AdvLanguageSwitch/index.tsx:42]:
```suggestion
    const appLang = i18next?.language?.toUpperCase() || 'EN';
```

File: src/domain/Common/LanguageSwitch/index.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const LanguageSwitch = (): ReactElement => {
   useEffect(() => {
     // Prepare list of languages for selector removing 'en-US' used
     // for development purposes only.
-    const appLang = i18next.language.toUpperCase();
+    const appLang = i18next?.language ? i18next.language.toUpperCase() : 'EN';
```
Comment (tataihono) [inline src/domain/Common/LanguageSwitch/index.tsx:33]:
```suggestion
    const appLang = i18next?.language?.toUpperCase() || 'EN';
```

File: src/domain/Common/LanguageSwitch/index.tsx [hunk 3]
```diff
@@ -52,7 +52,7 @@ const LanguageSwitch = (): ReactElement => {
 
   const newLangSelected = (option: Option): void => {
     // Change current language for app UI.
-    const appLang = i18next.language.toUpperCase();
+    const appLang = i18next?.language ? i18next.language.toUpperCase() : 'EN';
```
Comment (tataihono) [inline src/domain/Common/LanguageSwitch/index.tsx:55]:
```suggestion
    const appLang = i18next?.language?.toUpperCase() || 'EN';
```

**Review by tataihono — CHANGES_REQUESTED** [review]
you can just use ? to chain the entire lot.

**Pull request merged**